editor/test/integration/save_data_test.clj

```diff
@@ -612,6 +616,11 @@
         (is (= [2.0 2.0 2.0] (g/node-value embedded-go :scale)))
         (is (= [2.0 2.0 2.0] (g/node-value referenced-go :scale)))))
 
+    (testing "font"
+      (let [extra-characters-font (project/get-resource-node project "/silently_migrated/extra_characters.font")]
+        (is (= " !\"#$%&'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghijklmnopqrstuvwxyz{|}~åäö"
+               (g/node-value extra-characters-font :characters)))))
+
```

Comment:
Test that the silent migration moves any `:extra-characters` into `:characters`. The `åäö` came from `:extra-characters`.